src/components/layouts/AnyPageLayout.tsx

```diff
@@ -0,0 +1,11 @@
+export function AnyPageLayout({
+	children,
+}: {
+	children: JSX.Element | JSX.Element[];
+}) {
+	return (
+		<div className="px-6 md:px-32 lg:px-40 xl:px-52 pt-14 pb-12 md:pb-24 lg:pb-20">
```

Comment:
Ide lehet kéne még egy max-w-t beállítani, hogy szélesebb kijelzőkön se folyjon szét a content.

Reply:
és ha issue #50 -nél fixeljük egyben az összes oldalnál?

Reply:
Nekem az is jó